src/components/new-request/name-input.vue

```diff
@@ -92,6 +95,9 @@ export default class NameInput extends Vue {
     if (this.getErrors.includes('name')) {
       return ['Please enter a name to search for']
     }
+    if (this.getErrors.includes('long')) {
+      return ['Cannot exceed 150 characters']
```

Comment:
Instead of using a hardcoded value for the max characters, you could get it from a constant and share it with actions.ts

Reply:
done